### PR TITLE
simplify woven annotation adding

### PR DESCRIPTION
**Pull request by BrandonArp**
depends on the release of https://github.com/ArpNetworking/maven-javassist/pull/8

### CODE REVIEW

File: src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
  */
 package com.arpnetworking.commons.builder;
 
+import com.arpnetworking.commons.builder.annotations.WovenValidation;
```
Comment (vjkoskela) [inline src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java]: Should we update line 82 to specify it's only needed as a plugin dependency?

File: src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java [hunk 2]
```diff
@@ -299,22 +300,22 @@ public void process(final CtClass ctClass) {
...
-        if (annotationAttribute == null) {
-            annotationAttribute = new AnnotationsAttribute(classFile.getConstPool(), AnnotationsAttribute.visibleTag);
-            classFile.addAttribute(annotationAttribute);
-        }
-        javassist.bytecode.annotation.Annotation annotation = annotationAttribute.getAnnotation(PROCESSED_ANNOTATION_CLASS);
-        if (annotation == null) {
-            annotation = new javassist.bytecode.annotation.Annotation(PROCESSED_ANNOTATION_CLASS, classFile.getConstPool());
-        }
+
+        final javassist.bytecode.annotation.Annotation annotation =
```
Comment (vjkoskela) [inline src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java]: This code now has a direct dependency on Javassist (it was transitive before this afaik). Please update the dependency list in the class javadoc.

**Comment by vjkoskela** [comment]
Minor comments. Ship it.

File: src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java (hunk 2 above)
Comment (BrandonArp) [reply src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java]: Uh... take a look at lines ~20-28.  Note how those are not changed.

File: src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java (hunk 1 above)
Comment (BrandonArp) [reply src/main/java/com/arpnetworking/commons/builder/ValidationProcessor.java]: Noted

**Pull request merged**